frontend/controller/app/identity.js

```diff
@@ -388,7 +388,9 @@ export default (sbp('sbp/selectors/register', {
             })
           } else {
             try {
-              await sbp('chelonia/contract/sync', identityContractID)
+              if (navigator.onLine !== false) {
+                await sbp('chelonia/contract/sync', identityContractID)
+              }
```

Comment:
So... do we sync this once we do come back online?

Reply:
No, we don't explicitly. It'd be handled like any other contract by that point.